packages/camel_trail/camel_trail.gemspec

```diff
@@ -28,7 +28,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rake", "~> 13"
   s.add_development_dependency "rspec", "~> 3.0"
   s.add_development_dependency "rspec-rails", "~> 5.1.2"
-  s.add_development_dependency "rubocop-powerhome", "0.5.0"
```

Comment:
Why?

Reply:
not sure; potentially debugging? regardless, I restored it.